appservice/src/intTest/groovy/nu/yona/server/AbstractAppServiceIntegrationTest.groovy

```diff
@@ -67,4 +67,18 @@ abstract class AbstractAppServiceIntegrationTest extends Specification
 		int num = sequenceNumber++
 		return "$baseTimestamp$num"
 	}
-}
+
+	def assertDateTimeInLimits(dateTimeString, int rangeSeconds = 10, Date comparisonDateTime = null)
```

Comment:
More frameworks have e.g. `assertEquals(double expected, double actual, double epsilon)`, so I do not agree with the first part. However, it is a good idea to not allow the comparison without the third parameter.